app/main.py

```diff
@@ -2,0 +1,44 @@
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I\'m {self.name}")
+
+    def feed(self) -> str | int:
```

Comment:
The method `feed` is defined to return `str | int`, but it only returns an `int`. Consider updating the return type to `int` unless you plan to return a string in some cases.